core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -323,4 +323,6 @@ trait AuthenticationDirectiveProvider extends Spi {
   def authenticate(implicit transid: TransactionId,
                    authStore: AuthStore,
                    logging: Logging): AuthenticationDirective[Identity]
+
+  def namespaceIdentity(namespace: EntityName)(implicit transid: TransactionId, authStore: AuthStore): Future[Identity]
```

Comment:
Should we rename this to something more descriptive, like `identityByNamespace`? A little bit of ScalaDoc would help for context, i.e.:

```scala
/**
 * Retrieves an Identity based on a given namespace name.
 *
 * For use-cases of anonymous invocation (i.e. WebActions), 
 * we need to an identity based on a given namespace-name to 
 * give the invocation all the context needed.
 */
```

A little bit of ScalaDoc for `authenticate` would be great in the same vain.